src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -165,6 +165,14 @@ class ParticipantClinical extends React.Component {
     },
   ];
 
+  filterOutParentDiagnoses(diagnosis) {
+    if ((diagnosis?.mondo?.hits?.edges || []).length > 0) {
```

Comment:
why not putting "diagnosis?.mondo?.hits?.edges" in a const ? E.g: const diagnosisEdges = ...

Reply:
I thinks this class method can be extracted and put "outside" de scope of the class :+1: 


class MyClass {
    method1() {...}
}

VS 

const method1 = () => ....
class MyClass {...}